src/resources/js/commerce/gateway/toggler.js

```diff
@@ -157,7 +164,7 @@ tribe.tickets.commerce.gateway = tribe.tickets.commerce.gateway || {};
 			}
 			obj.toggler.showGateway( gateway );
 		});
-		obj.toggler.toggles.additional.addClass(obj.toggler.classes.toggleOpen);
+		obj.toggler.toggles.additional.addClass(obj.toggler.classes.toggleOpen).attr( 'aria-selected', 'true' );
```

Comment:
If I'm not mistaken we're applying this to the `containers` of all the additional gateways which are not the default, no? I may be reading this the wrong way.  The aria-selected should be applied to the "opener" and not the container.

Reply:
This is actually for the toggle button itself. The toggle button is the `opener`